src/bot.jl

```diff
@@ -8,24 +8,36 @@ const UStrings = Union{AbstractString,Regex,AbstractChar}
 Config object that holds the options and configuration for the SnoopCompile bot. This object is fed to the `@snoopi_bot`.
 
 # Arguments:
-- `packageName::String`
-- `subst::Vector{Pair{UStrings, UStrings}}` : to replace a packages precompile setences with another's package like `["ImageTest" => "Images"]`
-- `blacklist::Vector{UStrings}` : to remove some precompile sentences
+- `package_name::String`
 
-`const UStrings ==  Union{AbstractString,Regex,AbstractChar}` # every string like type that `replace()` has a method for.
+## optional:
+
+- `subst` : A vector of pairs of Strings (or RegExp) to replace a packages precompile setences with another's package like `["ImageTest" => "Images"]`.
+
+- `blacklist` : A vector of of Strings (or RegExp) to remove some precompile sentences
+
+- `os`: A vector of of Strings (or RegExp) to give the list of os that you want to generate precompile signatures for. Each element will call a `Sys.is\$eachos()` function.
```

Comment:
One disadvantage is that this doesn't clarify whether these are positional or keyword arguments. The usual way I might think of writing this kind of documentation would be

```julia
    BotConfig(package_name::AbstractString; subst = [], blacklist = [], os = [])

Construct a SnoopCompile bot configuration. `package_name` is the name of the package. You may supply one or more of the following keyword arguments:
...
```